e2e/cypress/integration/channel/react_to_last_message_shortcut_spec.js

```diff
@@ -0,0 +1,368 @@
+        });
+
+        // # This post is in Center, where reaction be added
+        cy.getLastPostId().then((lastPostId) => {
+            cy.get(`#${lastPostId}_message`).as('postInCenter');
+        });
+
+        // * Check if the emoji picker has the entered reaction to the message in the center
+        cy.get('@postInCenter').within(() => {
+            cy.findByLabelText('reactions').should('exist');
+            cy.findByLabelText('remove reaction smile').should('exist');
+        });
+
+        // * Check if the emoji picker has not entered reaction to the message in the RHS
+        cy.get('@postInRHS').within(() => {
+            cy.findByLabelText('reactions').should('not.exist');
+            cy.findByLabelText('remove reaction smile').should('not.exist');
+        });
+
+        cy.closeRHS();
+    });
+
+    it('Should also open emoji picker for last message with Mac keyboard', () => {
+        // # Making sure there is at least a message without reaction
+        cy.postMessage(MESSAGES.TINY);
+
+        // # Emulate react to last message shortcut when focus is on the center textbox by MAC keyboard
+        cy.get('#post_textbox', {timeout: TIMEOUTS.LARGE}).focus().type('{cmd}{shift}\\');
+
+        // * Check that emoji picker is opened
+        cy.get('#emojiPicker').should('exist').within(() => {
+            // # Search for an emoji and add it to message
+            cy.findByPlaceholderText('Search Emoji').type('smile{enter}');
+        });
+
+        // * Check if the emoji picker has the entered reaction to last message
+        cy.getLastPostId().then((lastPostId) => {
+            cy.get(`#${lastPostId}_message`).within(() => {
+                cy.findByLabelText('reactions').should('exist');
+                cy.findByLabelText('remove reaction smile').should('exist');
+            });
+        });
+    });
+
+    it('Should not open emoji picker when channel is new and empty or achived', () => {
+        // # Visit the new empty channel
+        cy.visit(`/ad-1/channels/${newChannel.name}`);
+
+        // * Check that there are no posts except you joined message
+        cy.findAllByTestId('postView').should('have.length', 1);
+
+        // # Emulate react to last message shortcut
+        cy.get('body').type('{ctrl}{shift}\\');
+
+        // * Check that emoji picker is not opened for empty channel
+        cy.get('#emojiPicker').should('not.exist');
+
+        // # Post a message to channel
+        cy.postMessage(MESSAGES.TINY);
+
+        // # Archive the channel after posting a message
+        cy.apiDeleteChannel(newChannel.id);
+
+        // # Emulate react to last message shortcut
+        cy.get('body').type('{ctrl}{shift}\\');
+
+        // * Check that emoji picker is not opened
+        cy.get('#emojiPicker').should('not.exist');
+
+        // # Close the archived channel
+        cy.findByText('Close Channel').should('exist').click();
+    });
+});
```

Comment:
I'm really happy to see these tests.  That's awesome!

Reply:
Thank you @saturninoabril learned it from you itself